packages/chevrotain/src/parse/parser/traits/perf_tracer.ts

```diff
@@ -13,7 +12,7 @@ export class PerformanceTracer {
   traceInitIndent: number
 
   initPerformanceTracer(config: IParserConfig) {
-    if (has(config, "traceInitPerf")) {
+    if (config.traceInitPerf) {
```

Comment:
I turned this into a check for truthiness, so that it doesn't run with `traceInitPerf: false`. I assume that's the intended behavior.